update how to persist enabling ipv6 forwarding across reboot (#48572)
kubernetes/website: content/en/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md

following the steps from [Installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).

For each server that you want to use as a {{< glossary_tooltip text="node" term_id="node" >}},
make sure it allows IPv6 forwarding. 

### Enable IPv6 packet forwarding {#prerequisite-ipv6-forwarding}

To check if IPv6 packet forwarding is enabled:

```bash
sysctl net.ipv6.conf.all.forwarding
```

> If the output is `net.ipv6.conf.all.forwarding = 1` it is already enabled. If the output is 
> `net.ipv6.conf.all.forwarding = 0`, then follow next steps.

Reviewer: ```suggestion
If the output is `net.ipv6.conf.all.forwarding = 1` it is already enabled.
Otherwise it is not enabled yet.
```
